cardano_node_tests/tests/tests_conway/test_pparam_update.py

```diff
@@ -1175,6 +1186,122 @@ def _check_state(state: dict):
         if fin_voted_votes.drep:
             governance_utils.check_vote_view(cluster_obj=cluster, vote_data=fin_voted_votes.drep[0])
 
+        # Check for deposit return
+        deposit_amt = cluster.conway_genesis["govActionDeposit"]
+        enact_deposit_returned = cluster.g_query.get_stake_addr_info(
+            pool_user_lg.stake.address
+        ).reward_account_balance
+        assert (
+            enact_deposit_returned
+            == init_return_account_balance + deposit_amt * submitted_proposal_count
+        ), "Incorrect return account balance"
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_update_pparams_deposit_return_after_expiry(
```

Comment:
Can the existing `test_pparam_update` test be extended to check the deposit return? We aim to limit the overall testrun runtime.
There's a number of pparam proposals that expire in the test.

Reply:
Initial thought was it might not be clear that we are waiting for gov_action_lifetime in the existing tests as multiple proposals and epochs were waited in between.

The latest update test_pparam_update has been extended to check the deposit return for both.